### PR TITLE
Problem: failed to bump crate `uniffi` to `0.19` (fix #451)

### LINKED ISSUE #451: Problem: failed to bump crate `uniffi` to `0.19`
**Describe the bug**

Failed bump PRs:
https://github.com/crypto-com/defi-wallet-core-rs/pull/449
https://github.com/crypto-com/defi-wallet-core-rs/pull/442
https://github.com/crypto-com/defi-wallet-core-rs/pull/450

**To Reproduce**

**Expected behavior**

Succeed to bump `uniffi` to `0.19`.

### PR DESCRIPTION
**Pull request by stevenatcrypto**
Close #451 

### Summary

1. Bump `uniffi_bindgen` to version `0.19.1`.
2. Update README.
3. Fix to always try to install `uniffi_bindgen` (with specified version) to avoid using the cache one in CI (if the version is right, will not re-install).

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [x] Have you read the [CONTRIBUTING.md](https://github.com/crypto-com/defi-wallet-core-rs/blob/main/CONTRIBUTING.md)?
- [x] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [x] Have you rebased your work on top of the latest main? 
- [x] Have you checked your code compiles? (`cargo build`)
- [x] Have you included tests for any non-trivial functionality?
- [x] Have you checked your code passes the unit tests? (`cargo test`)
- [x] Have you checked your code formatting is correct? (`cargo fmt -- --check --color=auto`)
- [x] Have you checked your basic code style is fine? (`cargo clippy`)
- [x] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`cargo audit`)
- [x] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [x] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-com/defi-wallet-core-rs/blob/main/CHANGELOG.md)?
- [x] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-com/defi-wallet-core-rs/blob/main/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-com/defi-wallet-core-rs/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: .github/workflows/android-build.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
-      - run: command -v uniffi-bindgen || cargo install uniffi_bindgen --version 0.18.0
+      - run: cargo install uniffi_bindgen --version 0.19.1
```
Comment (tomtau) [inline .github/workflows/android-build.yml:45]: will it be always reinstalling now?

**Review by tomtau — APPROVED** [review]
there's a breaking change: https://github.com/mozilla/uniffi-rs/blob/v0.19.1/CHANGELOG.md#%EF%B8%8F-breaking-changes-%EF%B8%8F but I guess it didn't affect android/iOS?

File: .github/workflows/android-build.yml (hunk 1 above)
Comment (stevenatcrypto) [reply .github/workflows/android-build.yml:45]: No. I try it on local, it only displayed a message as below:
```
% cargo install uniffi_bindgen --version 0.19.1
Ignored package `uniffi_bindgen v0.19.1` is already installed, use --force to override
```

**Review by damoncro — APPROVED** [review]

**Comment by stevenatcrypto** [comment]
> there's a breaking change: https://github.com/mozilla/uniffi-rs/blob/v0.19.1/CHANGELOG.md#%EF%B8%8F-breaking-changes-%EF%B8%8F but I guess it didn't affect android/iOS?

Seems no. We do not custom or have any code of [BindingGeneratorConfig](https://github.com/mozilla/uniffi-rs/blob/ef0813bf3ecc03981064cabe7151cd220f812062/uniffi_bindgen/src/lib.rs#L217).

**Pull request merged**